packages/emails/src/components/ManageLink.tsx

```diff
@@ -16,9 +16,10 @@ export function ManageLink(props: { calEvent: CalendarEvent; attendee: Person })
   const hasBookingLink = Boolean(bookingLink);
   const isRecurringEvent = props.calEvent.recurringEvent;
   const shouldDisplayRescheduleLink = Boolean(hasRescheduleLink && !isRecurringEvent);
+  const isTeamMember = props.calEvent.team?.members.some((member) => props.attendee.email === member.email);
 
   if (
-    (isOriginalAttendee || isOrganizer) &&
+    (isOriginalAttendee || isOrganizer || isTeamMember) &&
```

Comment:
For round robin, CTA was only being added for Round Robin Host and not for other fixed host.

For collective, CTA was only being included for a single host.